Lecture5.tex

```diff
@@ -434,7 +434,7 @@ \subsection{Some Background on Linear Programming}
 
 \begin{align*}
 \mathop {\min }\limits_{x \ge 0,Ax \ge c} {b^T}x &= \mathop {\min }\limits_{x \ge 0} \mathop {\max }\limits_{y \ge 0} \left\{ {{b^T}x + {y^T}(c - Ax)} \right\}\\
- &= \mathop {\max }\limits_{y \ge 0} \mathop {\min }\limits_{x \ge 0} \left\{ {{c^T}y + {x^T}(b - Ay)} \right\} = \mathop {\max }\limits_{y \ge 0,Ay \le b} {c^T}x.
+ &= \mathop {\max }\limits_{y \ge 0} \mathop {\min }\limits_{x \ge 0} \left\{ {{c^T}y + {x^T}(b - {A^T}y)} \right\} = \mathop {\max }\limits_{y \ge 0,{A^T}y \le b} {c^T}y.
```

Comment:
1. Moving from the first line to the second, transpose was taken, but A was forgotten.
2. Final maximization is on c^Ty, not x.